azure/README.md

```diff
@@ -0,0 +1,57 @@
+# Microsoft Azure
+
+This small setup can be used to provision Flatcar nodes on [Azure](https://azure.microsoft.com/).
+
+Create a `terraform.tfvars` file that lists your preferences. Like this one:
+
+```
+cluster_name            = "mycluster"
+machines                = ["mynode"]
+ssh_keys                = ["ssh-rsa AA... me@mail.net"]
+flatcar_stable_version  = "x.y.z"
+```
+
+You can resolve the latest Flatcar Stable version with this shell command:
+
+```
+curl -sSfL https://stable.release.flatcar-linux.net/amd64-usr/current/version.txt | grep -m 1 FLATCAR_VERSION_ID= | cut -d = -f 2
```

Comment:
Can this be done also for arm64 images? I'm asking, because I noticed that below you run terraform with `ARM_SUBSCRIPTION_ID` and other env vars.

Reply:
Or maybe ARM here means Azure Resource-or-something Manager-or-something?

Reply:
Yes, ARM is not about Arm ISA

Reply:
The curl command is just used to query the latest Stable release to find the right image. There are no arm64 images nor instances available.